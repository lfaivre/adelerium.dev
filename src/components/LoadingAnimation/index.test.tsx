import React from 'react';
import renderer from 'react-test-renderer';

import { LoadingAnimation } from '.';

describe('LoadingAnimation', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<LoadingAnimation />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
