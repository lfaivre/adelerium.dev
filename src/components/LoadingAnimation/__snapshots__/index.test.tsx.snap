// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingAnimation renders correctly 1`] = `
<div
  className="loading-ripple css-d1eztt"
>
  <div />
  <div />
</div>
`;
